### ISSUE TITLE
Data generation with new torch.fft

### ISSUE THREAD

**Issue by lingda-li** [zongyi-li/fourier_neural_operator#70]
Thanks for your great work!

I'm trying to generate new training data for ns using `data_generation/navier_stokes/ns_2d.py`. Because I'm using the latest PyTorch so I try to change the old fft code to the new ones.

For example, `w_h = torch.rfft(w0, 2, normalized=False, onesided=False)` is changed to `w_h = torch.fft.fft2(w0)`,
```
        temp = q[...,0].clone()
        q[...,0] = -2*math.pi*k_y*q[...,1]
        q[...,1] = 2*math.pi*k_y*temp
        q = torch.irfft(q, 2, normalized=False, onesided=False, signal_sizes=(N,N))
```
is changed to
```
        temp = q.real.clone()
        q.real = -2*math.pi*k_y*q.imag
        q.imag = 2*math.pi*k_y*temp
        q = torch.fft.ifft2(q, norm="forward")
```

However I found out these numbers will quickly explode to nan after several steps, maybe caused by not doing normalization. I suspect my fft code conversion is incorrect. Could you please share your thoughts on this?

Thank you very much!

**Comment by lingda-li** [comment]
That will be great! My email is lildmh@gmail.com

Thanks a ton!

**Comment by ssssww0905** [comment]
> I suspect it's the normalization in fft and ifft (norm="forward" / norm="backward"). I can send you the code for pytorch 1.8 or higher (torch.fft.ftt) if you can send me an email?

Hello, could you please send me a copy of the code of the new version(torch.fft.)? My email is wwxxyy0905@gmail.com. Thanks a lot!

**Comment by foss98** [comment]
Hi @zongyi-li could you also please send me the updated code for data generation as I have been having trouble adapting it to later pytorch versions too. Email: cfos3120@uni.sydney.edu.au

**Comment by hanrong96** [comment]
Hi @zongyi-li, could you also please send me the updated code for data generation too? Email: [hanrong@nudt.edu.cn] Thanks!
